src/main/java/org/folio/rest/impl/PurchaseOrderHelper.java

```diff
@@ -150,6 +131,27 @@
       );
   }
 
+
+  public CompletableFuture<JsonObject> validateIfPOProtectedFieldsChanged(CompositePurchaseOrder compPO,
+      JsonObject compPOFromStorage) {
+    if (!compPOFromStorage.getString(WORKFLOW_STATUS)
```

Comment:
Just a reason to think...
If converting an order into a composite order before `if`, the conditional expression becomes more readable, but I have doubts whether it is correct to convert and validate the order in the one method.

Reply:
I didn't want to convert it to an object because JsonObject is needed all the way for verification, it will just be an over head to convert back and forth, so leaving it as is

Reply:
You still can use jsonObject from parameter for verification